### PR TITLE
DEV: Delete old personal message settings

### PR DESCRIPTION
**Pull request by martin-brennan**
Followup to e62e93f83a77adfa80b38fbfecf82bbee14e12fe

Both enable_personal_messages and min_trust_to_send_messages
have been deprecated for a long time now, they can be deleted.

### CODE REVIEW

**Comment by martin-brennan** [comment]
Turning back into draft because there are a couple of places this is still used.

**Comment by nattsw** [comment]
Sorry it me.. Did you get to check if https://github.com/discourse/discourse-new-topic-dropdown/blob/a511d566cc473614e96983e4804d2cc47170db37/javascripts/discourse/initializers/initialize-composer-toggle-hijack.js#L136 is still in use?

**Comment by martin-brennan** [comment]
@nattsw I swear new things come up every time that search is run, it's like whack-a-mole 😞 PRs for both here:

* https://github.com/discourse/discourse-new-topic-dropdown/pull/8
* https://github.com/discourse/discourse-composer-dropdown-pm/pull/2

File: config/site_settings.yml [hunk 1]
```diff
@@ -831,10 +831,6 @@ posting:
       ja: 4
       zh_CN: 4
       zh_TW: 4
-  enable_personal_messages:
```
[the remark this answers is not in the document]
Comment (tgxworld) [reply config/site_settings.yml:834]: We'll also have to clean up the site settings rows in the DBs if present.

File: config/site_settings.yml [hunk 2]
```diff
@@ -1592,10 +1588,6 @@ trust:
   min_trust_to_allow_self_wiki:
     default: 3
     enum: "TrustLevelSetting"
-  min_trust_to_send_messages:
```
Comment (tgxworld) [inline config/site_settings.yml:1595]: Same as https://github.com/discourse/discourse/pull/21381/files#r1201358964

**Comment by martin-brennan** [comment]
@tgxworld I have added a commit to delete the rows now

**Pull request merged**